SU-SWS/stanford_basic
# Fa fixup

# READY FOR REVIEW

# Summary
- Update to latest Decanter and compile
- Change how FontAwesome is being added to project to help with issues being inherited down the line (Decanter will no longer have FA and neither with sub-themes. Rather, stanford_basic will host FA for use down the chain)

# Needed By (Date)
- N/A

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that latest Decanter has been added (a good test: new normalize is present and FontAwesome is gone)
3. Verify that FontAwesome is being added to the project as needed

# Affected Projects or Products
- stanford_basic
- su_humsci_theme
- decanter

# Associated Issues and/or People
- N/A

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
